src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -230,5 +237,18 @@ bool MulticopterLandDetector::_get_landed_state()
 	return true;
 }
 
+float MulticopterLandDetector::get_takeoff_throttle()
+{
+	if (_control_mode.flag_control_manual_enabled && _control_mode.flag_control_position_enabled) {
+		return 0.9f;
+	}
+
+	if (_control_mode.flag_control_manual_enabled && _control_mode.flag_control_attitude_enabled) {
+		return 0.15f;
+	}
+
+	return 0.0f;
+}
```

Comment:
0.15 was there, it's minimal throttle to start a takeoff in manual flight mode.
I chose 0.9 to be useful when taking off in position mode. You don't accidentally hit this throttle point and it gives you a good takeoff velocity such that you come quickly of the ground. You are right, defined constants would be better. I'll fix that.